extension/content.css

```diff
@@ -94,43 +94,38 @@ readerease-sentence {
 #readingDisplayContainer {
 	position: fixed; 
 	top: 2%;
-	left: 40%;
-	width: 20%;
+	left: 45%;
+	width: 10%;
 	opacity: 0;
 	transition: all 2s ease 0s;
 	z-index: 1000;
 }
 #timerContainer {
 	width: 100%;
-	border: 1px solid rgba(0, 220, 255, 0.5);
-	box-shadow: 0 0 1pt 1pt rgba(0, 220, 255, 0.5);
-	background-color: rgba(15, 19, 28, 0.9);
-	border-radius: 5px 5px 15% 15%;
+	box-shadow: 0pt 1pt 1pt 1pt rgba(104, 104, 104, 0.801);
+	background-color: rgba(255, 255, 255, 0.95);
+	border-radius: 5px;
 }
 #speedContainer {
+	display: none;
```

Comment:
Now doesn't show as default; shows when changed and when in auto mode

Reply:
Nice!